### PR TITLE
rename folder and revert manifest.json

### PR DESCRIPTION
**Pull request by mattreex**
Rename diw environment and restore manifest

### CODE REVIEW

File: diwv1.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -1,173 +0,0 @@
...
-    "project_id": "dev",
-    "auth_filter_field": "gen3_resource_path",
-    "auth_filter_node_types": [
-      "subject"
-    ]
-  },
-  "jupyterhub": {
-    "enabled": "yes"
-  },
-  "global": {
```
Comment (diw2) [inline diwv1.planx-pla.net/manifest.json:159]: could you please add the global block?

File: diw.planx-pla.net/manifest.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "notes": [
+    "This is the dev environment manifest",
+    "That's all I have to say"
+  ],
+  "jenkins": {
+    "autodeploy": "yes"
+  },
+  "versions": {
+    "arborist": "quay.io/cdis/arborist:master",
+    "arranger": "quay.io/cdis/arranger:master",
```
Comment (diw2) [inline diw.planx-pla.net/manifest.json:11]: you can remove arranger and the arranger block. we are not using this service in gen3

File: diw.planx-pla.net/manifest.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    "arranger-dashboard": "quay.io/cdis/arranger-dashboard:master",
+    "arranger-adminapi": "quay.io/cdis/arranger-server:master",
```
Comment (diw2) [inline diw.planx-pla.net/manifest.json:13]: also arranger dashboard adminapi thanks

File: diwv1.planx-pla.net/manifest.json [hunk 4]
```diff
@@ -1,173 +0,0 @@
...
-    "pidgin": "quay.io/cdis/pidgin:master",
-    "portal": "quay.io/cdis/data-portal:master",
-    "revproxy": "quay.io/cdis/nginx:1.17.6-ctds-1.0.1",
-    "sheepdog": "quay.io/cdis/sheepdog:master",
-    "sower": "quay.io/cdis/sower:master",
-    "spark": "quay.io/cdis/gen3-spark:master",
-    "tube": "quay.io/cdis/tube:master",
-    "dashboard": "quay.io/cdis/gen3-statics:master"
-  },
-  "arborist": {
```
Comment (diw2) [inline diwv1.planx-pla.net/manifest.json:29]: this arborist block is required

File: diwv1.planx-pla.net/manifest.json [hunk 5]
```diff
@@ -1,173 +0,0 @@
...
-    "deployment_version": "2"
-  },
-  "indexd": {
```
Comment (diw2) [inline diwv1.planx-pla.net/manifest.json:32]: i think you want to add indexd block too

**Comment by mattreex** [comment]
Sorry, I used the cdis-manifest.bak file's manifest.json to restore. Should we just keep everything instead and just change the name?

File: diw.planx-pla.net/manifest.json [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  "arborist": {
+    "deployment_version": "2"
+  },
+  "indexd": {
+    "arborist": "true"
+  },
+  "global": {
+    "environment": "devplanetv1",
+    "hostname": "diwv1.planx-pla.net",
```
Comment (diw2) [inline diw.planx-pla.net/manifest.json:29]: diw.planx-pla.net

File: diw.planx-pla.net/manifest.json [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    "revproxy_arn": "arn:aws:acm:us-east-1:707767160287:certificate/c676c81c-9546-4e9a-9a72-725dd3912bc8",
+    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/datadictionary/develop/schema.json",
+    "portal_app": "dev",
+    "kube_bucket": "kube-diwv1-gen3",
+    "logs_bucket": "logs-diwv1-gen3",
```
Comment (diw2) [inline diw.planx-pla.net/manifest.json:34]: "kube_bucket": "kube_bucket.devplanetv1.gen3",
    "logs_bucket": "logs-devplanetv1-gen3",

**Comment by diw2** [comment]
diwv1  was not created for this namespace. Therefore there were few fields won't apply to diw namespace. It was an independent cluster and we destroyed it due to high bills.

**Pull request merged**